src/pages/App.tsx

```diff
@@ -1,5 +1,6 @@
 import ErrorBoundary from 'components/ErrorBoundary'
 import Loader from 'components/Icons/LoadingSpinner'
+import { IPFSSubpathRedirect } from 'components/IPFSSubpathRedirect'
```

Comment:
```suggestion
import { IpfsSubpathRedirect } from 'components/IpfsSubpathRedirect'
```
Will also need to change filename if you accept this change.